vncviewer/PlatformPixelBuffer.cxx

```diff
@@ -34,6 +34,7 @@
 #include <rdr/Exception.h>
 
 #include "PlatformPixelBuffer.h"
+#include <stdlib.h>
```

Comment:
Please try to maintain the header sorting/grouping. In this case this should be up higher with `<assert.h>`.